src/lib.rs

```diff
@@ -389,13 +389,13 @@ impl<S: Storage + Send + Sync + 'static> SelfEncryptor<S> {
         if file_size <= MAX_IN_MEMORY_SIZE as u64 {
             sequencer = Sequencer::as_vector();
         } else {
-            sequencer = Sequencer::as_mmap();
+            sequencer = Sequencer::as_mmap().unwrap();
```

Comment:
Another nice PR, this unwrap though is probably not good to leave. I think this may mean pushing Results up, which is OK and better than unwrap.